clicache/src/impl/AssemblyInfo.cpp.in

```diff
@@ -26,11 +26,11 @@ using namespace System::Security::Permissions;
 // set of attributes. Change these attribute values to modify the information
 // associated with an assembly.
 //
-[assembly:AssemblyTitleAttribute("GemFire.NETCache")];
-[assembly:AssemblyDescriptionAttribute("GemFire Native Client .NET Cache")];
+[assembly:AssemblyTitleAttribute("Geode.NETCache")];
+[assembly:AssemblyDescriptionAttribute("Geode Native Client .NET Cache")];
 [assembly:AssemblyConfigurationAttribute("")];
-[assembly:AssemblyCompanyAttribute("Pivotal Software, Inc.")];
-[assembly:AssemblyProductAttribute("GemFire.NETCache")];
+[assembly:AssemblyCompanyAttribute("Apache Software Foundation")];
+[assembly:AssemblyProductAttribute("Geode.NETCache")];
```

Comment:
Some of these fields should actually be filled in by the CMake variables that fill in the .rc file. Can you write up a separate ticket to do this?

Reply:
Done, GEODE-5251.